features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/config/api-manager.xml

```diff
@@ -760,5 +760,11 @@
     <JWTAuthentication>
         <EnableSubscriptionValidationViaKeyManager>false</EnableSubscriptionValidationViaKeyManager>
     </JWTAuthentication>
-
+    <CertificateReLoaderConfiguration>
+        <Period>10</Period>
+    </CertificateReLoaderConfiguration>
+    <MutualSSL>
+        <ClientCertificateHeader>X-WSO2-CLIENT-CERTIFICATE</ClientCertificateHeader>
+        <DisableClientCertificateValidation>false</DisableClientCertificateValidation>
```

Comment:
shall we change this to enable to match with `apimgt.mutual_ssl.enable_client_validation`?